alpaka/include/alpaka/intrinsic/IntrinsicUniformCudaHipBuiltIn.hpp

```diff
@@ -100,7 +100,7 @@ namespace alpaka
                     std::int64_t value)
                 -> std::int32_t
                 {
-                    return __ffsll(value);
+                    return __ffsll(static_cast<unsigned long long>(value));
```

Comment:
This might work for HIP, but `__ffsll` in CUDA takes a `long long int` as argument, so now there is a narrowing conversion in CUDA.

Reply:
HIP also provide a `long long` version I will fix it more clean in https://github.com/alpaka-group/alpaka/pull/1049/files